README.md

```diff
@@ -2,41 +2,210 @@
 [![Build Status Master](https://travis-ci.org/HearstAT/ensemble.svg?branch=master)](https://travis-ci.org/HearstAT/ensemble)
 [![Code Climate](https://codeclimate.com/github/HearstAT/ensemble/badges/gpa.svg)](https://codeclimate.com/github/HearstAT/ensemble) [![Test Coverage](https://codeclimate.com/github/HearstAT/ensemble/badges/coverage.svg)](https://codeclimate.com/github/HearstAT/ensemble/coverage) [![Issue Count](https://codeclimate.com/github/HearstAT/ensemble/badges/issue_count.svg)](https://codeclimate.com/github/HearstAT/ensemble)
 
-Things you may want to cover:
+## Requirements
 
-* Ruby version
+* Ruby Version 2.3.x
+* Docker and Docker Compose (Latest)
+* Postgresql (tested with 9.5 and 9.6)
+* sqlite
 
-This has been developed/tested on Ruby 2.3.x
+### Ruby
 
```

Comment:
I'd probably make RVM a subsection here rather than it being the primary, so if folks (like me) want to add documentation for rbenv or just system ruby setup.